networkx/readwrite/gml.py

```diff
@@ -153,12 +153,12 @@ def read_gml(path, label='label', destringizer=None):
 
     label : string, optional
         If not None, the parsed nodes will be renamed according to node
-        attributes indicated by ``label``. Default value: ``'label'``.
+        attributes indicated by `label`. Default value: `'label'`.
```

Comment:
The double backticks around the string literal `'label'` are important. In other words, this should remain ``` ``'label'`` ```.